keras/models.py

```diff
@@ -659,6 +659,31 @@ def evaluate_generator(self, generator, val_samples,
         return self.model.evaluate_generator(generator,
                                              val_samples)
 
+    def predict_generator(self, generator, val_samples,
+                           **kwargs):
+        '''Generate predictions for the input samples from a data generator. 
+        The generator should return the same kind of data as accepted by
+        `predict_on_batch`.
+
+        Arguments:
+            generator:
+                generator yielding input samples
+            val_samples:
+                total number of samples to generate from `generator`
+                before returning.
+
+        # Returns
+            A Numpy array of predictions.
+        '''
+        if 'verbose' in kwargs:
```

Comment:
This method is new, so it has never had a `verbose` argument. This check was for backwards compatibility. Here it isn't required. You can just remove `kwargs` management altogether.